test/functional/tools/filter_data_table.xml

```diff
@@ -0,0 +1,45 @@
+<tool id="filter_data_table" name="filter data table" version="0.1.0">
+    <description>Filter on datatable entries</description>
+    <command><![CDATA[
+       echo $dynamic_select > '$output'
+    ]]></command>
+    <inputs>
+        <param name="dynamic_select" type="select">
+            <!-- initialize options from file (contains all entries that could be added to the data table) -->
+            <options from_file="../test/functional/tool-data/test_file.tsv">
```

Comment:
That's a security issue and should not work.

Reply:
Do you have an idea how to specify another dir during tests, eg `test/functional/tool-data/`?

I guess we can check if `os.path.abspath(path)` startswith the `tool-data` dir... could do this in another PR.

Reply:
It's fine to do this in another PR, but we shouldn't merge this before we sorted that out.

Reply:
Also there is a `safe_contains` function in galaxy.util you can use

Reply:
ae4c40145cb9398fed05fc8cc5664b3e9522bd81 contains a possible solution. Any comments?